less/home-page.less

```diff
@@ -1,31 +1,86 @@
 .home {
+  max-width: 1400px;
```

Comment:
I think your container is wider than the container in the design file and is wider than the header image. it's causing your image to be too small relative to the following sections.